### DISCUSSION TITLE
Missing some HTTP options

### DISCUSSION THREAD

**Discussion by KemplerPP** [axios/axios#5427] [Ideas]
Hi,
Some request options mentioned in [Node.js documentation](https://nodejs.org/docs/latest-v16.x/api/http.html#httprequesturl-options-callback) are not yet available in Axios, for instance `maxHeaderSize`.
To support it in Axios, someone could add it to the [http-adapter](https://github.com/axios/axios/blob/abd2cb83a3bc3be81438763eb50efb809b2b5a94/lib/adapters/http.js#L386):
`options.maxHeaderSize = config.maxHeaderSize`
Would it be possible for a contributor to add this option? What about using more options in a more generic way (spread operator)?
I think that it should be a minor change for all of v0, v1, v2.
If no contributor wishes to take this task, could I contribute myself?
I thank you in advance.

**Comment by jasonsaayman** [comment]
you may create a PR for this and i will review it asap
